### PR TITLE
Added alternative titles

### LINKED ISSUE #397: Include more languages for show alternative titles
**Enhancement:**

Alternative titles nowadays have more options than just `English`, `Japanese` and `Synonyms`.
New languages that are available: `German`, `Spanish`, `French`.

[Example](https://myanimelist.net/anime/38000)

**How to fix**

* Add methods similar to `getTitleEnglish` in the `AnimeParser`, just change method name and change the xpath with the new languages.
* Add nullable properties for the new languages in the model `Anime`.

**Version**

`v3.0.0-beta.9`

### PR DESCRIPTION
**Pull request by nerg4l**
This PR adds logic for parsing alternative titles. Not sure if this is the best logic for doing this so I'm open for suggestions.

Resolves #397

### CODE REVIEW

File: src/Model/Manga/Manga.php [hunk 1]
```diff
@@ -189,6 +194,7 @@ public static function fromParser(MangaParser $parser): Manga
         $instance->titleEnglish = $parser->getMangaTitleEnglish();
         $instance->titleSynonyms = $parser->getMangaTitleSynonyms();
         $instance->titleJapanese = $parser->getMangaTitleJapanese();
+        $instance->alternativeTitles = $parser->getAlternativeTitles();
```
Comment (irfan-dahir) [inline src/Model/Manga/Manga.php:197]: @nerg4l  Should this be `titleAlternatives` in order to match the naming order of the rest (`titleEnglish`, `titleSynonyms`, `titleJapanese`)?

Which would get auto serialized to `title_alternatives` by the REST API.

**Review by irfan-dahir — COMMENTED** [review]
The approach looks good to me. 

It does repeat values for title english and japanese. I guess we can gracefully deprecate that in the next major release? I'm thinking along the lines of grouping all titles into one object for the next major release which should handle the scaling here.

**Comment by nerg4l** [comment]
I think we could already group the titles and expose a `titles` property.

**Comment by irfan-dahir** [comment]
@nerg4l This would be a major version change though?

**Comment by nerg4l** [comment]
`getTitleAlternatives()` returns an array but to be able to call `getDefault()` it has to return a collection object.

**Comment by nerg4l** [comment]
I was planning to rename the exposed property to `titles`.

**Comment by nerg4l** [comment]
Tomorrow I will have time.

**Pull request merged**